charts/argo-controller/templates/config.yaml

```diff
@@ -13,6 +13,13 @@ kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}-config
 data:
+  {{- if .Values.logs.enabled }}
+  artifactRepository: |
+    archiveLogs: true
+    gcs:
+      bucket: {{ .Values.logs.bucket }}
+      keyFormat: {{ .Values.logs.prefix }}/key
```

Comment:
Question for my understanding - Is this the keyFormat predefined or something we are setting here to later match on the monster-deploy side?

Reply:
It's a predefined key, used to set the paths in the bucket.

I think this will need to use the workflow / pod names as part of the path, otherwise all the logs will layer on top of each other.

Reply:
The example [here](https://github.com/argoproj/argo/blob/master/docs/configure-artifact-repository.md#google-cloud-storage-gcs) says this string can use Argo variables like `{{workflow.name}}` and `{{pod.name}}`